sdk/nodejs/restAPI.ts

```diff
@@ -131,7 +131,7 @@ export interface RestAPIArgs {
      *
      * Either `swaggerString` or `routes` must be specified.
      */
-    routes?: inputs.RouteArgs[];
+    routes?: pulumi.Input<inputs.RouteArgs>[];
```

Comment:
This is another change in types.

Reply:
Hmm, yes, which means it doesn't look to be limited to Go 🤔

Reply:
It's also changed in python

Reply:
Ah marshal bug then, thanks that'll probably make this _much_ easier to fix.